wntr/network/controls.py

```diff
@@ -2095,6 +2102,56 @@ def run_control_action(self):
                 control_action.run_control_action()
         else:
             raise RuntimeError('control actions called even though if-then statement was False')
+    
+    def update_condition(self, condition, verbose = False):
+        """Update the controls condition in place"""
```

Comment:
Please add docstrings to each method to document input parameters and remove `verbose`.  The ability to set the logger level should be sufficient to control verbosity.

Reply:
I've improved the docstrings and removed the verbose argument in commit: 6e57605.